Tiltfile

```diff
@@ -96,12 +108,7 @@ path_port_forwards = ["6060:6060"]
 # be routed through Envoy Proxy.
 if MODE == "path_with_auth":
```

Comment:
Suggestion: add a TODO to remove `MODE`: all deployments should use GUARD (which can allow a single, specific endpoint id by default).

Reply:
> Suggestion: add a TODO to remove `MODE`: all deployments should use GUARD (which can allow a single, specific endpoint id by default).

Agreed, I'm definitely in support of this.

Reply:
I've just removed `MODE` altogether since it was unneeded.